<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-E6SRQGTN" failures="1" tests="5" name="Test" time="6.118" errors="0" timestamp="2022-09-30T09:26:08 IST">
  <testcase classname="section16.Annotations" name="OpeningBrowser" time="6.034"/>
  <testcase classname="section16.Annotations" name="activities" time="0.004"/>
  <testcase classname="section16.Annotations" name="fail" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at section16.Annotations.fail(Annotations.java:83)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- fail -->
  <testcase classname="section16.Annotations" name="FlightBooking" time="0.001"/>
  <testcase classname="section16.Annotations" name="FlightCancel" time="0.001"/>
  <testcase classname="section16.Annotations" name="calc" time="0.0">
    <ignored/>
  </testcase> <!-- calc -->
</testsuite> <!-- Test -->
